releasenotes/notes/0.3/0.3-summary-8bdd0273e78a9795.yaml

```diff
@@ -0,0 +1,116 @@
+        class.
+      - Adds new solver methods accessible through :func:`.solve_ode` and :func:`.solve_lmde`.
+features:
+  - |
+    Adds support for Python 3.10. Starting with this release
+    Qiskit Dynamics now supports running with Python 3.7 through 3.10.
+  - |
+    The  :meth:`.Solver.solve` method can now run multiple simulations in a single call by
+    specifying one or more of the ``t_span``, ``y0``, and ``signals``
+    arguments as lists of valid single inputs. This supports specifying all as a list of the
+    same length (with a different value for each to be used in each simulation), or a subset
+    can be specified with lists of the same length, with the other arguments giving only a single
+    value (to be used repeatedly while varying the other parameters). Results are returned as a
+    list of the results of the individual simulations. The list of simulations is performed
+    in a serial loop.
+  - |
+    The :class:`.Solver` class has been upgraded to enable Qiskit Pulse schedule simulation.
+    Configuration information for interpeting pulse schedules must be provided at instantiation,
+    and schedules can be solved by passing them via the ``signals`` argument to
+    :meth:`.Solver.solve`.
+  - |
+    Adds :class:`.ArrayPolynomial` for representing multi-variable array-valued polynomials.
+  - |
+    Adds function :func:`.solve_lmde_perturbation` for computing Dyson series and Magnus
+    expansion terms for solutions to linear matrix differential equations.
+  - |
+    Adds data container class :class:`.PerturbationResults` for storing and accessing the
+    results of :func:`.solve_lmde_perturbation`.
+  - |
+    Adds classes :class:`.DysonSolver` and :class:`.MagnusSolver` implementing perturbation
+    expansion-based fixed-step solvers for linear matrix differential equations.
+  - |
+    Adds support for solvers from the diffrax package:
+    https://github.com/patrick-kidger/diffrax. To call a diffrax solver, pass it as the
+    ``method`` kwarg to :func:`solve_lmde`. For example::
+
+      from diffrax import Dopri5
+      from qiskit-dynamics import solve_ode
+
+      results = solve_ode(rhs=rhs, t_span=t_span, y0=y0, method=Dopri5())
+  - |
+    Adds new methods ``'lanczos_diag'`` and the JAX version ``'jax_lanczos_diag'``
+    to `solve_lmde`. This is an approximate matrix-exponentiation method, based
+    on diagonalizing an approximate projection of the generator to a small subspace
+    (the Krylov Subspace), obtained via the Lanczos algorithm.  The size of this
+    subspace is determined by the parameter ``k_dim``, which controls the accuracy of the
+    approximation. The method is recommended for sparse systems with large dimension.
+  - |
+    Updates the ``Solver.solve`` method to preserve dimensionality data of the input
+    states when producing output states. E.g. if ``y0`` is a ``Statevector``, the output
```

Comment:
Should this be a bug fix rather than feature?

Reply:
Changed to a fix.